test/gtest/uct/test_md.cc

```diff
@@ -371,6 +387,112 @@ UCS_TEST_P(test_md, reg_multi_thread) {
     pthread_join(thread_id, NULL);
 }
 
+/*
+ * Test that ib md does not create umr region if 
+ * UCT_MD_MEM_ACCESS_REMOTE_ATOMIC is not set
+ */
+
+void test_md::ib_md_umr_check(void *rkey_buffer, bool amo_access) {
+
+    ucs_status_t status;
+    size_t size = 8192;
+    void *buffer = malloc(size);
+    ASSERT_TRUE(buffer != NULL);
+
+    uct_mem_h memh;
+    status = uct_md_mem_reg(pd(), buffer, size, 
+                            amo_access ? UCT_MD_MEM_ACCESS_ALL :  UCT_MD_MEM_ACCESS_RMA,
```

Comment:
it does not, but ACCESS_ATOMIC should be sufficient to make atomic key valid,
however what happens if UMR is not supported on a machine? looks like this test is not appropriate
maybe better to change p2p tests to register memory with minimal required access flags instead of using ALL in the base class